discovery/module.go

```diff
@@ -307,6 +307,29 @@ func (m *Module) Services() []ServiceDefinition {
 	return result
 }
 
+// GetServiceActivation is a Discovery Client function that retrieves the activation status of a service for a DID.
+// See interface.go for more information.
+func (m *Module) GetServiceActivation(_ context.Context, serviceID string, subjectDID did.DID) (bool, *vc.VerifiablePresentation, error) {
+	refreshTime, err := m.store.getPresentationRefreshTime(serviceID, subjectDID)
+	if err != nil {
+		return false, nil, err
+	}
+	if refreshTime == nil {
+		return false, nil, nil
+	}
```

Comment:
this tests if the client is managing the subjectDID's status on the discovery server, which is not in the API description